docker-compose.dev.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.8'
+services:
+  pic-sure-auth-service:
+    build:
+        context: .
+        dockerfile: ./pic-sure-auth-services/dev.Dockerfile
+    environment:
+      JAVA_OPTS: "-Xmx1024m -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
+      # Note: To use this you will need to set ENVIRONMENT variables for the application.properties
+    ports:
+      - "8090:8090" # Application port
+      - "5005:5005" # Debug port
```

Comment:
Don't expose this by default. You could make this an env var

Reply:
This is a dev docker-compose file intended for those who are willing to try to deploy locally.